### PR TITLE
NoLeadingImportSlashFixer - Add space if needed

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4438

### CODE REVIEW

File: tests/Fixer/Import/NoLeadingImportSlashFixerTest.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function provideFixCases()
             ],
             [
                 '<?php
-                use/*1*/A\B;
+                use/*1*/ A\B;
```
Comment (kubawerlos) [inline tests/Fixer/Import/NoLeadingImportSlashFixerTest.php]: Well, no, the original version looks correct.

File: src/Fixer/Import/NoLeadingImportSlashFixer.php [hunk 2]
```diff
@@ -65,13 +66,31 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
...
     }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     */
+    private function removeLeadingImportSlash(Tokens $tokens, $index)
+    {
+        if (
+            $tokens->getNextNonWhitespace($index) > $index + 1
```
Comment (kubawerlos) [inline src/Fixer/Import/NoLeadingImportSlashFixer.php]: What is the use of `getNextNonWhitespace`? I cannot see any test case for this.

File: src/Fixer/Import/NoLeadingImportSlashFixer.php [hunk 3]
```diff
@@ -65,13 +66,31 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
...
+            || $tokens->getPrevNonWhitespace($index) < $index - 1
```
Comment (kubawerlos) [inline src/Fixer/Import/NoLeadingImportSlashFixer.php]: `!$tokens[$index - 1]->isWhitespace()`?

File: tests/Fixer/Import/NoLeadingImportSlashFixerTest.php [hunk 4]
```diff
@@ -47,7 +47,7 @@ public function provideFixCases()
...
                 ',
                 '<?php
                 use/*1*/\A\B;
```
Comment (kubawerlos) [inline tests/Fixer/Import/NoLeadingImportSlashFixerTest.php]: One more test for `use\/*1*/A\B;`?

**Comment by SpacePossum** [comment]
I don't care enough about the PR to fix everything in it

**Comment by SpacePossum** [comment]
thanks for the review @kubawerlos , I think I processed all your suggestions, can you have a look again?

**Pull request merged**